### ISSUE TITLE
Feature request: Naming consistency

### ISSUE THREAD

**Issue by SlimeDog** [LandlordPlugin/LandLord#7]
You know the drill. :)

- Spigot page: `Landlord 2`
- download: `landlord-latest-4.350-all.jar`
- plugin.yml: `Landlord`
- plugin subdirectory: `Landlord` (per plugin.yml)

So it would be nice if the downloaded file was `Landlord-4.350.jar`

**Comment by RainbowDashLabs** [comment]
The latest in the jar is required, because we differ between a legacy and a latest version of the plugin which work on different versions.

I think the name of the plugin is a legacy reasons since this plugin is a revival of the old landlord and should work as a drop in replacement. Thats why the page is Landlord 2. Renaming the page is not a real option.

Renaming the jar to Landlord-latest-4.350.jar would be an option, but still the `all` classifier indicates that the file is build by our ci and contains the dependencies.
Without the all it would be the reference jar builded by our ci. So naming matters.
I don't think that's the jar naming is so important tbh as long as the names of the jar are consistent, which they are.

This whole plugin is very historical grown.

tldr;
Page -> Landlord 2 because of revive of Landlord (1)
download -> because of ci. name helps dev to know which file the user downloaded
plugin name -> Without two to be used as an drop in replacement for the old landlord.

thb I think this is already pretty organized and well defined in our possibilities xD

But maybe @SirYwell and @Aurelien30000 have another opinion on this.

**Comment by Aurelien30000** [comment]
Nomenclature is just adapted imo, large admittedly. Users need to be able to differentiate modules and versions clearly. 
The only possible inconsistency might by the classifier _(i.e. all)_, honestly others are required, ``name-module-version``. Most of plugins follow this model.

**Comment by SlimeDog** [comment]
Thanks for the reasoning. No worries. You know my predilections. :)
